core/clingutils/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ include_directories(${LLVM_INCLUDE_DIRS} ${CLING_INCLUDE_DIRS})
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/res ${CMAKE_CURRENT_SOURCE_DIR}/../foundation/res)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CLING_CXXFLAGS}")
 
+# This is to avoid warnings coming from GCC 7 in llvm/src/include/llvm/ADT/DenseMap.h:1010
```

Comment:
A bit late to answer, but yes, that's what the CDash link above indicates.
Since this commit was already added, I will change later as Vassil suggested to only affect the necessary files rather than all of clingutils.